core/src/test/java/org/apache/iceberg/TableTestBase.java

```diff
@@ -349,6 +349,35 @@ void validateSnapshot(Snapshot old, Snapshot snap, long sequenceNumber, DataFile
     validateSnapshot(old, snap, (Long) sequenceNumber, newFiles);
   }
 
+  Snapshot commit(Table tbl, SnapshotUpdate snapshotUpdate, String branch) {
```

Comment:
Minor: we avoid unnecessary abbreviations, like `tbl` in place of `table`. It's often more awkward to type the shortened version and it makes the code harder to read. I think in this case both are true. `t` and `b` right after one another are hard to type, but I it's easy to type `tab`.

Reply:
good point, yeah in this case it makes sense just to use the actual name

Reply:
I forgot why I did tbl, it was because table was already used and was failing the hidden field checkstyle. I updated to ignore the checkstyle warning in favor of more readable variable names.